Sources/Common/Store/Migrations/SDKAlwaysInitializedMigration.swift

```diff
@@ -0,0 +1,148 @@
+import Foundation
+
+/**
+ Migration done for SDK always initizlized change we made to the SDK: https://github.com/customerio/issues/issues/10797
+
+ Before this change to the SDK, the SDK sandboxed the SDK data by siteid. The background queue and profile id for profile identified to name a few examples.
+ After this change, the SDK stores all of it's SDK data in a single place. The SDK does not support using the SDK with multiple workspaces and therefore, no need to sandbox.
+
+ Run after the SDK has been initialized so we have a site id.
+ */
+public class SDKAlwaysInitializedMigration: BaseMigration {
+    let siteId: String
+
+    let sandboxedFileStorage: SandboxedSiteIdFileStorage
```

Comment:
should we drop these instances after the migration is completed? because i suspect them to be big in memory?